grafana/setup_grafana.go

```diff
@@ -79,7 +79,7 @@ func main() {
 		Database:  envParams["influxdb_database"],
 	}
 
-	grafanaURL := fmt.Sprintf("%s://%s:%s@localhost:%s", envParams["gf_server_protocol"], envParams["grafana_user"], envParams["grafana_passwd"], envParams["grafana_port"])
+	grafanaURL := fmt.Sprintf("%s://%s:%s@localhost:%s", envParams["gf_server_protocol"], envParams["gf_security_admin_user"], envParams["gf_security_admin_password"], envParams["gf_server_http_port"])
```

Comment:
The logic maybe like this:
In case we make the default value for `gf_security_admin_user` and `gf_security_admin_user` be empty string. 

* if `gf_security_admin_user` and `gf_security_admin_user` is not empty, i.e., user specify a `gf_security_admin_user` and `gf_security_admin_user` explicitly, we should use `gf_security_admin_user` and `gf_security_admin_user` to construct `grafanaURL`.
* otherwise, we fallback to the original way, i.e., we should use `grafana_user` and `grafana_passwd` to construct `grafanaURL`.